### PR TITLE
✨ [FEAT] Make email optional in Report form (refs #4085)

### PR DESCRIPTION
**Pull request by Chatewgne**
## Checklist

- [x] I have followed the guidelines in our [Contributing document](https://geotrek.readthedocs.io/en/latest/contribute/contributing.html)
- [x] My code respects the Definition of done available in the [Development section of the documentation](https://geotrek.readthedocs.io/en/latest/contribute/development.html#definition-of-done-for-new-model-fields)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes
- [x] I added an entry in the changelog file
- [x] My commits are all using prefix convention (emoji + tag name) and references associated issues
- [x] I added a label to the PR corresponding to the perimeter of my contribution
- [x] The title of my PR mentionned the issue associated

### CODE REVIEW

File: geotrek/feedback/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -305,10 +311,17 @@ msgstr ""
...
+msgid "A new feedback has been sent."
+msgstr "Un problème a été signalé."
+
 #, python-format
 msgid "Comment : %(comment)s"
 msgstr "Commentaire : %(comment)s"
 
+#, python-format
+msgid "Trek : %(name)s"
+msgstr "Randonnée : %(name)s"
```
Comment (submarcos) [inline geotrek/feedback/locale/fr/LC_MESSAGES/django.po:323]: jusque là, Trek est traduit en "Itinéraire". C'est voulu le "randonnée" ?

File: geotrek/feedback/models.py [hunk 2]
```diff
@@ -236,7 +235,10 @@ def comment_text(self):
         return html.unescape(self.comment)
 
     def send_report_to_managers(self, template_name="feedback/report_email.txt"):
-        subject = _("Feedback from {email}").format(email=self.email)
+        if self.email:
+            subject = _("Feedback from {email}").format(email=self.email)
```
Comment (submarcos) [inline geotrek/feedback/models.py:239]: pour respecter la maniere de mettre des variables dans les trads, utilise la notation `%()s`

output = _("Today is %(month)s %(day)s.") % {"month": m, "day": d}

File: geotrek/feedback/templates/feedback/report_email.txt [hunk 3]
```diff
@@ -1,11 +1,18 @@
 {% load i18n l10n %}
 {% autoescape off %}
-{% blocktrans with email=report.email %}{{email}} has sent a feedback.{% endblocktrans %}
+{% if report.email %}
+    {% blocktrans with email=report.email %}{{email}} has sent a feedback.{% endblocktrans %}
```
Comment (submarcos) [inline geotrek/feedback/templates/feedback/report_email.txt:4]:
```suggestion
    {% blocktrans with email=report.email %}{{ email }} has sent a feedback.{% endblocktrans %}
```

File: geotrek/feedback/templates/feedback/report_email.txt [hunk 4]
```diff
@@ -1,11 +1,18 @@
...
+{% else %}
+    {% trans "A new feedback has been sent." %}
+{% endif %}
 
 {% trans "Activity" %} : {{report.activity}}
 {% if report.comment %}{% blocktrans with comment=report.comment_text|striptags %}Comment : {{comment}}{% endblocktrans %}{% endif %}
 {% trans "Category" %} : {{report.category}}
 {% trans "Problem magnitude" %} : {{report.problem_magnitude}}
+{% if report.related_trek %}{% blocktrans with name=report.related_trek.name|striptags %}Trek : {{name}}{% endblocktrans %}{% endif %}
```
Comment (submarcos) [inline geotrek/feedback/templates/feedback/report_email.txt:13]:
```suggestion
{% if report.related_trek %}{% blocktrans with name=report.related_trek.name|striptags %}Trek : {{ name }}{% endblocktrans %}{% endif %}
```

File: geotrek/feedback/templates/feedback/report_email.txt [hunk 5]
```diff
@@ -1,11 +1,18 @@
...
+{% trans "Report" %} {% if report.eid|length %}{{report.eid}}{%else%}{{report.id}}{%endif%}
```
Comment (submarcos) [inline geotrek/feedback/templates/feedback/report_email.txt:14]:
```suggestion
{% trans "Report" %} {% if report.eid|length %}{{ report.eid }}{% else %}{{ report.id }}{% endif %}
```

File: geotrek/feedback/locale/fr/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -305,10 +311,17 @@ msgstr ""
 msgid "%(email)s has sent a feedback."
 msgstr "%(email)s a signalé un problème."
 
+msgid "A new feedback has been sent."
```
Comment (marcantoinedupre) [inline geotrek/feedback/locale/fr/LC_MESSAGES/django.po:314]: Erreur de traduction

**Review by submarcos — APPROVED** [review]

**Pull request merged**